massdriver-application-gcp-cloud-run/main.tf

```diff
@@ -23,6 +23,9 @@ resource "google_cloud_run_service" "main" {
       service_account_name = module.application.id
       containers {
         image = var.container_image
+        ports {
+          container_port = var.container_port
```

Comment:
Cloud Run encourages the user to listen to the `PORT` env-var. This is to support container images that do not (like ones made from cloud functions buildpacks).